.github/workflows/notify-mm-blog.yml

```
@@ -0,0 +1,32 @@
+name: notify-mm-blog
+on:
+  push:
+    paths:
+      - 'site/content/blog/**'
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - shell: bash
+        run: |
+          set -x
+          git diff --name-only --diff-filter=A HEAD~1 -- 'site/content/blog/*.md' > file.log
+          FILEADDED=$(cat file.log)
+          echo $FILEADDED
+          if [ -z "$FILEADDED" ]
+          then
+            echo "nothing added in the .md"
+            exit 0
+          else
+            FILENAME=$(cat $FILEADDED | grep slug | cut -d':' -f 2 | sed 's/^ *//g')
+            echo $FILENAME
+            TITLE=$(cat $FILEADDED | grep title | cut -d':' -f 2 | sed 's/^ *//g')
+            echo "{\"username\":\"GitHub Action Notify\",\"icon_url\":\"https://www.mattermost.org/wp-content/uploads/2016/04/icon.png\",\"attachments\":[{\"author_name\":\"mattermost.dev\",\"author_link\":\"https://mattermost.dev\",\"fallback\":\"New Blog Post! Check out [here](https://developers.mattermost.com/blog/)\",\"color\":\"good\",\"title\":\""$TITLE"\",\"text\":\"New Blog Post! Check out [here](https://developers.mattermost.com/blog/$FILENAME)\"}]}" > mattermost.json
```

Terrific! Just a minor prose suggestion:
```suggestion
            echo "{\"username\":\"GitHub Action Notify\",\"icon_url\":\"https://www.mattermost.org/wp-content/uploads/2016/04/icon.png\",\"attachments\":[{\"author_name\":\"mattermost.dev\",\"author_link\":\"https://mattermost.dev\",\"fallback\":\"New Blog Post! Check out [here](https://developers.mattermost.com/blog/)\",\"color\":\"good\",\"title\":\""$TITLE"\",\"text\":\"New Blog Post! Check it out [here](https://developers.mattermost.com/blog/$FILENAME).\"}]}" > mattermost.json
```